lms/templates/staff_problem_info.html

```diff
@@ -42,7 +44,7 @@ <h2>${_("{platform_name} Content Quality Assessment").format(platform_name=setti
       <label for="${element_id}_xqa_entry">${_("Comment")}</label>
       <input tabindex="0" id="${element_id}_xqa_entry" type="text" placeholder="${_('comment')}">
       <label for="${element_id}_xqa_tag">${_("Tag")}</label>
-      <span style="color:black;vertical-align: -10pt">${_('Optional tag (eg "done" or "broken"):') + '&nbsp; '}      </span>
+      <span style="color:black;vertical-align: -10pt">${_('Optional tag (eg "done" or "broken"):') + HTML('&nbsp; ')}      </span>
```

Comment:
Can't that just be moved outside of the expression?
```
${_('Optional tag (eg "done" or "broken"):')}&nbsp;      
```

Reply:
Done